smqtk_relevancy/utils/read_write_lock.py

```diff
@@ -0,0 +1,486 @@
+"""
+LICENCE
+-------
+Copyright 2013-2020 by Kitware, Inc. All Rights Reserved. Please refer to
+LICENSE.TXT for licensing information, or contact General Counsel,
+Kitware, Inc., 28 Corporate Drive, Clifton Park, NY 12065.
+"""
+
+from collections import deque
```

Comment:
I think this is only used in that distance kernel class that can be removed, so this file may also be removed I think.